net/Socket.cpp

```diff
@@ -894,22 +918,29 @@ std::shared_ptr<Socket> ServerSocket::accept()
             std::shared_ptr<Socket> _socket = createSocketFromAccept(rc);
 
 #if !MOBILEAPP
-            char addrstr[INET6_ADDRSTRLEN];
-
-            const void *inAddr;
-            if (clientInfo.sin6_family == AF_INET)
+            if (clientInfo.sin6_family == AF_UNIX)
             {
-                auto ipv4 = (struct sockaddr_in *)&clientInfo;
-                inAddr = &(ipv4->sin_addr);
+                _socket->setClientAddress("/");
```

Comment:
I wonder if we could have something more useful for the filtering & access control logic here? which is IIRC string based; could we use SO_PEERCRED to get a uid at least out if it?

Reply:
I set the client address to "/" because a function already exists and handles the Unix socket in this way

https://github.com/CollaboraOnline/online/blob/52e3d4cb7ca39008fd272b6e9d26d74640dbcc33/net/Socket.cpp#L998-L1008